pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/graphql_helpers.go

```diff
@@ -40,47 +46,6 @@ func getRepoData(
 	return nil, errors.New("not able to get repo count")
 }
 
-//func getBranchData(
-//	ctx context.Context,
-//	client graphql.Client,
-//	owner string,
-//	ownertype string,
-//	// here we use a pointer to a string so that graphql will receive null if the
-//	// value is not set since the after: $repoCursor is optional to graphql
-//	repoCursor *string,
-//	branchCursor *string,
-//	// since we're using a interface{} here we do type checking when data
-//	// is returned to the calling function
-//) (interface{}, error) {
-//	if ownertype == "user" {
-//		data, err := getUserBranchData(ctx, client, owner, repoCursor, branchCursor)
-//		if err != nil {
-//			return nil, err
-//		}
-//		return data, nil
-//	} else if ownertype == "org" {
-//		data, err := getOrgBranchData(ctx, client, owner, repoCursor, branchCursor)
-//		if err != nil {
-//			return nil, err
-//		}
-//		return data, nil
-//	}
-//
-//	return nil, errors.New("not able to get repo count")
-//}
-
-//func (ghs *githubScraper) getRepoBranchInformation(ctx context.Context, client graphql.Client, owner string, ownertype string) error {
```

Comment:
This was deleted because I started to add it as part of the genqlient refactor and left it in as a comment, but the current implementation is still using the function declared on line 89 in github_scraper.go until it gets removed as part of the continuous refactor to genqlient.